example/bufferCopy/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ if(NOT TARGET alpaka::alpaka)
     endif()
 endif()
 
+if (alpaka_USE_MDSPAN STREQUAL "OFF")
```

Comment:
I would prefer to not including the `CMakeLists.txt` of the `bufferCopy` example, if mdspan is not enabled:

`example/CMakeLists.txt`:
```cmake
if (NOT alpaka_USE_MDSPAN STREQUAL "OFF")
  add_subdirectory("bufferCopy/")
endif()
```

I think a warning should be fixable for your current build and enabling an optional feature to fix the warning sounds not like a good solution.
Also my IDE is constantly display all warnings and want's that I fix it ;-)

Reply:
If I do that, you will get a compile error then when you compile the bufferCopy example standalone, and not from the alpaka build tree. That is, you run cmake inside `example/bufferCopy`. I don't think this is better.